annotation-utils/src/main/java/edu/isi/vista/annotationutils/PushAnnotations.kt

```diff
@@ -0,0 +1,145 @@
+package edu.isi.vista.annotationutils
+
+import com.jcraft.jsch.JSch
+import com.jcraft.jsch.Session
+import edu.isi.nlp.parameters.serifstyle.SerifStyleParameterFileLoader
+import java.io.File
+import org.eclipse.jgit.api.Git
+import org.eclipse.jgit.api.ResetCommand
+import org.eclipse.jgit.api.TransportConfigCallback
+import org.eclipse.jgit.diff.DiffEntry
+import org.eclipse.jgit.lib.ObjectId
+import org.eclipse.jgit.revwalk.RevCommit
+import org.eclipse.jgit.revwalk.RevWalk
+import org.eclipse.jgit.transport.*
+import org.eclipse.jgit.treewalk.CanonicalTreeParser
+import org.eclipse.jgit.util.FS
+import java.text.SimpleDateFormat
+import java.util.*
+
+/**
+ * The goal of this program is to export the annotations, save the restored
+ * documents, compare with the previous commit to see if any changes have been
+ * made, and if so, push those changes to the repo.
+ *
+ * The program takes one argument: a file containing these parameter values:
+ *
+ * <ul>
+ *     <li> {@code exportAnnotationsParams}: the location of the parameter file meant for {@code ExportAnnotations.kt} </li>
```

Comment:
One advantage of Kotlin's doc tool over Javascript is that you can use regular Markdown, so *I think* you can just surround these with backticks rather than needing the `@code`s.

Reply:
Replaced `@code`s with backticks